### PR TITLE
Rosa perez

### PR DESCRIPTION
**Pull request by paintedlbird7**

### CODE REVIEW

**Comment by RaDeleon** [comment]
Everything looks good! Nice job on the sprint and getting MVP done since I know this was a hard week.

**Comment by RaDeleon** [comment]
Just noticed it later but console.log error with Proptypes

Cards.js line:21
  author:PropTypes.number.isRequired  

Author is a number? Or a string?? Its going to be a 'string' 

Just change to string and you will get rid of the errors. Small mistake so just fix that and you're fine. Besides this great job on passing this sprint.

File: ANSWERS.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+every Component follows a cycle from when it's created and mounted on the DOM to when it is unmounted and destroyed:  Initialization, Mounting, Updating, Unmounting
+
+
+ Explain the details of a Higher Order Component?
+
+HO is for reusing components.  It is a function that takes a component and returns a new component.
+
+ What are three different ways to style components in React? Explain some of the benefits of each.
+
+ To reused a style button as a dev you don't have to create mutliple styling, just one and reuse it, saves time
```
Comment (PhilHart23) [inline ANSWERS.md]: 5/5

File: lambdatimes/src/components/Content/Card.js [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+    PropTypes.shape({
+  headline: PropTypes.string.isRequired,
+  author:PropTypes.number.isRequired,
+  img: PropTypes.string.isRequired,
+  tab: PropTypes.string.isRequired,
+    })
+  )
+  }
+
 export default Card;
```
Comment (PhilHart23) [inline lambdatimes/src/components/Content/Card.js]: proptypes look good!

File: lambdatimes/src/components/Content/Content.js [hunk 3]
```diff
@@ -48,8 +79,18 @@ export default class Content extends Component {
...
+
+        selectTabHandler={this.changeSelected}
+
+        />
+        <Cards 
+        cards={this.filterCards()} />
+
       </div>
     );
   }
```
Comment (PhilHart23) [inline lambdatimes/src/components/Content/Content.js]: Nice work!

File: lambdatimes/src/components/TopBar.js [hunk 4]
```diff
@@ -21,4 +24,83 @@ const TopBar = () => {
...
+font-size: 11px;
+font-weight: bold;
+`
+const Topbardiv10 = styled.div`
+.top-bar .container .container-right span {
+cursor: pointer;
+`
+
+
 export default TopBar;
```
Comment (PhilHart23) [inline lambdatimes/src/components/TopBar.js]: Styled components look good!

**Pull request merged**